ci/jenkins/test-unit-integration.sh

```diff
@@ -0,0 +1,203 @@
+do
+key="$1"
+
+case $key in
+    --kubeconfig)
+    KUBECONFIG_PATH="$2"
+    shift 2
+    ;;
+    --workdir)
+    WORKDIR="$2"
+    shift 2
+    ;;
+    --registry)
+    DOCKER_REGISTRY="$2"
+    shift 2
+    ;;
+    --coverage)
+    COVERAGE=true
+    shift
+    ;;
+    --codecov-token)
+    CODECOV_TOKEN="$2"
+    shift 2
+    ;;
+    -h|--help)
+    print_usage
+    exit 0
+    ;;
+    *)    # unknown option
+    echoerr "Unknown option $1"
+    exit 1
+    ;;
+esac
+done
+
+if [[ "$WORKDIR" != "$DEFAULT_WORKDIR" && "$KUBECONFIG_PATH" == "$DEFAULT_KUBECONFIG_PATH" ]]; then
+    KUBECONFIG_PATH=$WORKDIR/.kube/config
+fi
+
+# If DOCKER_REGISTRY is non null, we ensure that "make" commands never pull from docker.io.
+NO_PULL=
+if [[ ${DOCKER_REGISTRY} != "" ]]; then
+    NO_PULL=1
+fi
+export NO_PULL
+
+function run_codecov { (set -e
+    flag=$1
+    file=$2
+    dir=$3
+    remote=$4
+    ip=$5
+
+    rm -f trustedkeys.gpg codecov
+    # This is supposed to be a one-time step, but there should be no harm in
+    # getting the key every time. It does not come from the codecov.io
+    # website. Anyway, this is needed when the VM is re-created for every test.
+    curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --no-default-keyring --keyring trustedkeys.gpg --import
+    curl -Os https://uploader.codecov.io/latest/linux/codecov
+    curl -Os https://uploader.codecov.io/latest/linux/codecov.SHA256SUM
+    curl -Os https://uploader.codecov.io/latest/linux/codecov.SHA256SUM.sig
+
+    # Check that the sha256 matches the signature
+    gpgv codecov.SHA256SUM.sig codecov.SHA256SUM
+    # Then check the integrity of the codecov binary
+    shasum -a 256 -c codecov.SHA256SUM
+
+    chmod +x codecov
+
+    if [[ $remote == true ]]; then
+        ${SCP_WITH_UTILS_KEY} codecov jenkins@${ip}:~
+        ${SSH_WITH_UTILS_KEY} -n jenkins@${ip} "cd antrea; ~/codecov -c -t ${CODECOV_TOKEN} -F ${flag} -f ${file} -C ${GIT_COMMIT} -r antrea-io/antrea"
+    else
+        ./codecov -c -t ${CODECOV_TOKEN} -F ${flag} -f ${file} -s ${dir} -C ${GIT_COMMIT} -r antrea-io/antrea
+    fi
+    rm -f trustedkeys.gpg codecov
+)}
+
+function run_integration {
+    flag=$1
+    VM_NAME="antrea-integration-0"
+    export GOVC_INSECURE=1
+    export GOVC_URL=${GOVC_URL}
+    export GOVC_USERNAME=${GOVC_USERNAME}
+    export GOVC_PASSWORD=${GOVC_PASSWORD}
+    VM_IP=$(govc vm.ip ${VM_NAME})
+    govc snapshot.revert -vm.ip ${VM_IP} initial
+    VM_IP=$(govc vm.ip ${VM_NAME}) # wait for VM to be on
+
+    set -x
+    if [[ ${flag} == "multicluster" ]];then
+      echo "===== Run Multi-cluster Integration tests ====="
+      # umask ensures that files are cloned with the correct permissions so that Docker caching can be leveraged
+      ${SSH_WITH_UTILS_KEY} -n jenkins@${VM_IP} "PATH=$PATH:/usr/local/go/bin && umask 0022 && git clone ${ghprbAuthorRepoGitUrl} antrea && cd antrea && git checkout ${GIT_BRANCH} && cd multicluster && NO_LOCAL=true make test-integration"
```

Comment:
`ghprbAuthorRepoGitUrl` environment will not be available because the daily jobs are not triggered by pull request.

I think it's not necessary to clone Antrea repo again on the remote node. When Jenkins triggers the job, the Antrea repo is cloned automatically and checkout locally. This shell script can assume that it's running under the Antrea git repo. It can do something like:

```
${SSH_WITH_UTILS_KEY} -n jenkins@${VM_IP} "mkdir -p antrea"
${SCP_WITH_UTILS_KEY} -r .git jenkins@${VM_IP}:~/antrea/
${SSH_WITH_UTILS_KEY} -n jenkins@${VM_IP} "cd antrea && git checkout -f && git clean -fd"
```

By the way, I couldn't find definitions of SSH_WITH_UTILS_KEY and SCP_WITH_UTILS_KEY in this file. Maybe you can refer to test-vmc.sh for the definitions, and put the definitions in the beginning of this file.